lib/enum.ml

```diff
@@ -45,27 +45,44 @@ module Str = struct
     in
     Ast_builder.Default.pstr_value ~loc Nonrecursive [value_description]
 
-  let from_string_case_from_constructor ~loc constructor =
+  let from_string_case_from_constructor ~loc ~raises constructor =
     let {pcd_name = {txt = value_name; _}; _} = constructor in
     let lhs = string_to_constant_pattern ~loc ~str:(String.lowercase_ascii value_name) in
-    let rhs = string_to_constructor_expression ~loc ~str:value_name in
+    let value_t = string_to_constructor_expression ~loc ~str:value_name in
+    let rhs =
+      if raises
```

Comment:
Although that's undocumented, our `if-then` style is overwhelmingly like:

```ocaml
if condition then
  ...
else
  ...
```

or

```ocaml
if
  condition
    ~on
    ~multiple
    ~lines
then
  ...
else
  ...
```

Reply:
I think it should be formatted correctly since we are reading this code now, regardless of a potential later use of ocamlformat.  Keeping it formatted consistently should also reduce the amount of code ocamlformat will have to change, which is also a good thing.